### PR TITLE
Fix Essential Builds 4.3 in TeamCity

### PR DESCRIPTION
**Pull request by AzuObs**
Set the neo4j docker image to the latest 4.3 image.

### CODE REVIEW

File: test-utils/src/main/java/apoc/util/TestContainerUtil.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public static Neo4jContainerExtension createEnterpriseDB(File baseDir, boolean w
                 .withFileSystemBind(canonicalPath, "/var/lib/neo4j/import") // map the "target/import" dir as the Neo4j's import dir
                 .withEnv("NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes")
 //                .withDebugger()  // uncomment this line for remote debbuging inside docker's neo4j instance
-                .withCreateContainerCmdModifier(cmd -> cmd.withMemory(1024 * 1024 * 1024l))
+                .withCreateContainerCmdModifier(cmd -> cmd.withMemory(2024 * 1024 * 1024l))
```
Comment (ncordon) [inline test-utils/src/main/java/apoc/util/TestContainerUtil.java:84]: Why this change?

**Comment by AzuObs** [comment]
[[Flaky test] StartupTest.check_basic_deployment is flaky on several 43 builds](https://trello.com/c/e5KzX8H7/112-flaky-test-startuptestcheckbasicdeployment-is-flaky-on-several-43-builds)
